VM/src/lstate.h

```diff
@@ -197,6 +197,7 @@ typedef struct global_State
     struct Table* mt[LUA_T_COUNT];                   // metatables for basic types
     TString* ttname[LUA_T_COUNT];       // names for basic types
     TString* tmname[TM_N];             // array with tag-method names
+    TString* lightuserdataname[LUA_LUTAG_LIMIT]; // names for tagged lightuserdata
```

Comment:
Please move this below `udatagc` to avoid large immediate offsets in native code for existing fields.